Signal/src/ViewControllers/MessageMetadataViewController.swift

```diff
@@ -585,7 +522,7 @@ class MessageMetadataViewController: OWSViewController {
         updateContent()
     }
 
-    private func MessageRecipientStateName(_ value: MessageRecipientState) -> String {
+    private func MessageRecipientStatusName(_ value: MessageRecipientStatus) -> String {
```

Comment:
nit: `MessageRecipientStatusName` -> `messageRecipientStatusName`

or maybe:`string(for messageRecipientStatus: MessageRecipientStatus)`

Reply:
Done.